Facebook.iOS/source/Facebook/ApiDefinition.cs

```diff
@@ -1810,6 +1814,40 @@ interface AppInviteDialogDelegate
 		void DidFail (AppInviteDialog appInviteDialog, NSError error);
 	}
 
+	// @interface FBSDKCameraEffectArguments : NSObject <FBSDKCopying, NSSecureCoding>
+	[BaseType (typeof (NSObject), Name = "FBSDKCameraEffectArguments")]
+	interface CameraEffectArguments : CoreKit.Copying, INSSecureCoding
+	{
+		// -(void)setString:(NSString *)string forKey:(NSString *)key;
+		[Export ("setString:forKey:")]
+		void SetString (string aString, string key);
+
+		// -(NSString *)stringForKey:(NSString *)key;
+		[Export ("stringForKey:")]
+		string StringForKey (string key);
+
+		// -(void)setArray:(NSArray<NSString *> *)array forKey:(NSString *)key;
+		[Export ("setArray:forKey:")]
+		void SetArray (string [] array, string key);
+
+		// -(NSArray *)arrayForKey:(NSString *)key;
+		[Export ("arrayForKey:")]
+		string [] ArrayForKey (string key);
+	}
+
+	// @interface FBSDKCameraEffectTextures : NSObject <FBSDKCopying, NSSecureCoding>
+	[BaseType (typeof (NSObject), Name = "FBSDKCameraEffectTextures")]
+	interface CameraEffectTextures : CoreKit.Copying, INSSecureCoding
+	{
+		// -(void)setImage:(UIImage *)image forKey:(NSString *)key;
+		[Export ("setImage:forKey:")]
+		void SetImage (UIImage image, string key);
+
+		// -(UIImage *)imageForKey:(NSString *)key;
+		[Export ("imageForKey:")]
+		UIImage ImageForKey (string key);
```

Comment:
Can we change this to GetImage?

Reply:
This might break consistency with the other types that still keep the *ForKey methods.
eg: ShareOpenGraphValueContaining

Reply:
I just noticed the SetImage right above with no forKey.